### PR TITLE
Price comparison

### PR DESCRIPTION
**Pull request by lauradumi**

### CODE REVIEW

File: app/controllers/stores_controller.rb [hunk 1]
```diff
@@ -1,15 +1,23 @@
 class StoresController < ApplicationController
+
```
Comment (sofofos) [inline app/controllers/stores_controller.rb]:
```suggestion
```

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,8 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   resources :my_items, controller: 'user_items', only: %i[index destroy]
+  # get 'my_items', to: 'user_items#index', as: 'my_items'
```
Comment (sofofos) [inline config/routes.rb]: cleanup commented lines
```suggestion
```

File: db/seeds.rb [hunk 3]
```diff
@@ -124,6 +124,7 @@
   end
   puts "\n"
 end
+
```
Comment (sofofos) [inline db/seeds.rb]:
```suggestion
```

**Review by sofofos — DISMISSED** [review]

File: app/views/user_items/index.html.erb [hunk 4]
```diff
@@ -15,5 +14,7 @@
         <% end %>
       </div>
     </div>
-  </div>
+
+ <%= link_to "Find best option", stores_results_path(list: @item_ids), class: "btn btn-dark text-center mt-10"%>
```
Comment (gabrieleli09) [inline app/views/user_items/index.html.erb]: <%= link_to "Find best deals!", stores_results_path(list: @item_ids), class: "btn btn-dark text-center mt-10"%>

**Review by gabrieleli09 — CHANGES_REQUESTED** [review]

**Review by gabrieleli09 — APPROVED** [review]
lgtm

**Pull request merged**